### ISSUE TITLE
Recent snaphshot of ETL servers don't send the humans/bots count

### ISSUE THREAD

**Issue by ETLegacyTeam** [etlegacy/etlegacy#664]
Author: @rmarquis (Spyhawk)
Date: 2014-11-23
Redmine Issue: 664, https://dev.etlegacy.com/issues/664

---

There are some online Legacy servers using recent -git snapshots (see screenshot). The x86_64 server uses pre.sdl2.570 while the red server uses 573, which is the most recent version as I type this bug report.

The humans/bots count is missing.

That feature has been introduced in 
"d09e28c7cf662414717bee46d5634adcb756cd53":https://github.com/etlegacy/etlegacy/commit/d09e28c7cf662414717bee46d5634adcb756cd53
and 
"1d0dad82a07c6f94dd5c9d68617a077d28f70d5e":https://github.com/etlegacy/etlegacy/commit/1d0dad82a07c6f94dd5c9d68617a077d28f70d5e.

---

Related issue(s): #271, #606

---

Related commit(s):
97defc00250496b4fa08b972ca3a20d081c21d39 - ui: fixed ETL server detection for humans and bots..

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2014-11-23T11:51:51Z

---

I can reproduce locally.

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2014-11-23T13:38:16Z

---

-I tried to use "041eeaf1":https://github.com/etlegacy/etlegacy/commit/041eeaf1, which is the last commit about the bots counter feature. The issue is the same, so that issue is not a regression. It seems the bot counter simply never worked with... git build? debug build? 64bit build?-

Disregard the above. The responsible commit is "7a49775":https://github.com/etlegacy/etlegacy/commit/7a497759dac4448969a2e01950801e6900a189bb.  The ET_VERSION identifier has been changed in #606, so there is no way to make the difference between 2.60b and ET:L server anymore.
